stablehlo/conversions/tosa/transforms/CMakeLists.txt

```diff
@@ -29,6 +29,7 @@ add_mlir_library(StablehloTOSATransforms
   DEPENDS
   StablehloTOSATransformsPassIncGen
   StablehloTOSAPDLLPatternsIncGen
+  StablehloOpsIncGen
```

Comment:
Could you also add `StablehloOps ` to `LINK_LIBS` below? Looks like this is a transitive dependency when including a file from that library, so probably best to explicitly call it our here. All other targets that include `StablehloOpsIncGen` also link `StablehloOps`

Reply:
Actually I'm working on a few minor patches, I'll loop this fix in!